tests/unit/changes_tests.py

```diff
@@ -465,14 +465,20 @@ def test_get_feed_using_doc_ids(self):
         self.assertSetEqual(set([x['id'] for x in changes]), expected)
         self.assertTrue(str(feed.last_seq).startswith('100'))
 
-    def test_invalid_argument(self):
```

Comment:
This test is no longer valid as passing foo='bar' is now accepted under ANY_ARG: ANY_TYPE.